tools/generate_events.py

```diff
@@ -112,6 +112,12 @@
 }
 {%- endif %}
 
+{%- if event.name == "host_status_updated" %}
```

Comment:
I wonder if we can define this based on `event.properties.info` like is being done for `cluster_id` above?

Reply:
+1